src/components/Auth/ResetPassword.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { useState, useCallback, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Form, Typography, Input, Button, notification } from 'antd';
+import styled from 'styled-components';
+import qs from 'query-string';
+
+import { useAuth } from '../../hooks';
+import { validatePassword } from './Auth';
+
+function ResetPassword() {
+  const [form] = Form.useForm();
+  const { authenticating, onPasswordReset, isPasswordResetted } = useAuth();
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState(' ');
+
+  const history = useHistory();
+
+  const q = qs.parse(document.location.search);
+  const token = q.token as string;
+  const login = (q.email as string) || '';
+
+  const handlePasswordChange = useCallback((e: any) => {
+    setPassword(e.target.value);
+  }, []);
+  const handleconfirmPasswordChange = useCallback((e: any) => {
+    setConfirmPassword(e.target.value);
+  }, []);
+
+  const resetSuccessfulNotification = () => {
```

Comment:
General function naming rule: it needs to have a verb describing the action it does. On the other hand variables are described with nouns. So:
`resetSuccessfulNotification` -> implies a variable
`notifySuccessfulReset` -> implies function